### ISSUE TITLE
Is this package maintained?

### ISSUE THREAD

**Issue by chmac** [alethes/meteor-pages#184]
Last commit dates back to June and there seem to be many unanswered issues. Wondering if @alethes is still working on it?

**Comment by brianmingus** [comment]
@alethes ping

**Comment by lowenf** [comment]
@alethes ping

**Comment by avalanche1** [comment]
not likely, based on @alethes github activity log

**Comment by GFargo** [comment]
This is unfortunate :disappointed: what next?

**Comment by luixal** [comment]
Ey, @aslagle , it would be great if your [reactive-table](https://github.com/aslagle/reactive-table) will meet this kind of backend efficiency :)

**Comment by sjmcdowall** [comment]
Guys -- you may want to look at the fork of this project -- https://atmospherejs.com/electronifiejs/pages -- it appears to be more active ..

**Comment by GFargo** [comment]
@sjmcdowall Awesome :+1: thanks for the heads up! Was hoping someone would force and continue development.

**Comment by luixal** [comment]
The project itself refers you to the original one for news and updates,
so... not so sure. Doc is just the same, or am I missing something?

**Comment by sjmcdowall** [comment]
yes, this one has had bug fixes applied and at least seems more "active" (or active at all).  <shrug>
